src/components/LightsOut.tsx

```
@@ -0,0 +1,146 @@
+
+      <div
+        className={styles.grid}
+        style={{ "--cols": gridSize } as React.CSSProperties}
+      >
+        {grid.map((row, i) =>
+          row.map((cell, j) => (
+            <div
+              key={`${i}-${j}`}
+              className={`${styles.cell} ${cell ? styles.on : styles.off}`}
+              onClick={() => toggle(i, j)}
+            />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)